labels.md

```diff
@@ -0,0 +1,220 @@
+status/future                   #fbca04     ?                                   
+type/1.0 cherry-pick            #c2e0c6     *                                   
+type/2.0 cherry-pick            #99d0ef     *                                   
+type/2.1 cherry-pick            #9bfff5     *
+type/3.0                        #ededed     *                                   
+type/3.0 cherry-pick            #297b99     *
+type/TEP                        #bfd4f2     type/TEP                            #1d76db
+type/bug                        #fc2929     type/bug                            #1d76db
+type/bug-fix                    #fc2929     type/bug fix                        #1d76db
+type/compatibility              #e99695     type/compatibility                  #1d76db
+type/duplicate                  #cccccc     status/closed dup                   #e6e6e6
+type/enhancement                #84b6eb     type/enhancement                    #1d76db
+type/invalid                    #e6e6e6     status/closed invalid               #e6e6e6
+type/investigation              #f46100     type/investigation                  #1d76db
+type/new-feature                #e2e876     * <type/enhancement>
+type/performance                #006b75     type/performance                    #1d76db
+type/regression                 #c41b2f     type/regression                     #1d76db
+type/suggestion                 #11b2c4     * <type/proposal>                     
+type/wontfix                    #95d3db     * <just close the issue>            
+v3.0                            #cb11f9     *                                   
+```
+
+## TiKV
+
+```
+C: Build                        #d1fad7     component/build                     #d1fad7
+C: Build-Time                   #d1fad7     component/build time                #d1fad7
+C: Copr                         #d1fad7     component/coprocessor               #d1fad7
+C: Doc                          #d1fad7     component/docs                      #d1fad7
+C: PD-Client                    #d1fad7     component/PD client                 #d1fad7
+C: Perf                         #d1fad7     component/perf                      #d1fad7
+C: Raft                         #d1fad7     component/raft                      #d1fad7
+C: RocksDB                      #d1fad7     component/RocksDB                   #d1fad7
+C: Server                       #d1fad7     component/server                    #d1fad7
+C: Storage                      #d1fad7     component/storage                   #d1fad7
+C: Test/Bench                   #d1fad7     component/test                      #d1fad7
+                                            component/bench                     #d1fad7
+C: TiKV-Client                  #d1fad7     component/TiKV client               #d1fad7
+C: TiKV-Ctl                     #d1fad7     component/TiKV ctl                  #d1fad7
+C: Titan                        #d1fad7     component/Titan                     #d1fad7
```

Comment:
Looks like we can merge C: RocksDB and C: Titan into C: Engine

Reply:
:+1: